setup.py

```diff
@@ -66,6 +66,7 @@ def has_environment_marker_platform_impl_support():
     python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
     install_requires=[
         'Twisted>=13.1.0',
+        'incremental',
```

Comment:
What do you think about adding a comment here, to explain why incremental is in install_requires?